third_party/conan/configs/linux/profiles/libfuzzer_base

```diff
@@ -0,0 +1,17 @@
+# This file is intended to be imported when building for oss-fuzz
+# Compiler and settings are auto-detected in that case, but we want to have
+# an upstream config file where settings can be changed.
+BASE_CFLAGS= -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -D_FORTIFY_SOURCE=2 -fstack-protector-all -fPIC
+BASE_CXXFLAGS= -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -fsized-deallocation -D_FORTIFY_SOURCE=2 -fstack-protector-all -fPIC
+BASE_LDFLAGS=-Wl,-z,relro,-z,now,-z,noexecstack -pthread
```

Comment:
Is this line correct? 
maybe:
```suggestion
BASE_LDFLAGS= -Wl,-z,relro,-z,now,-z,noexecstack,-pthread
```

Reply:
Yes the line is correct. Adding a comma to the -pthread option will break the build. As far as I know there is no -pthread-option that can be passed to `-Wl` or do you have different information?

Reply:
No I don't